docs/ConfigurationOptions/index.md

```diff
@@ -41,5 +41,6 @@ grate --connectionstring="Server=(localdb)\MSSQLLocalDB;Integrated Security=true
 | --baseline | - | **Baseline** - This instructs grate to mark the scripts as run, but not to actually run anything against the database. Use this option if you already have scripts that have been run through other means (and BEFORE you start the new ones). | 
 | --forceanytimescripts<br>--runallanytimescripts | false | **RunAllAnyTimeScripts** - This instructs grate to run any time scripts every time it is run even if they haven't changed. Defaults to false.
 | --dryrun | false | **DryRun** - This instructs grate to log what would have run, but not to actually run anything against the database.  Use this option if you are trying to figure out what grate is going to do. |
+| --restorefrompath | - | **RestoreFromPath** - This instructs grate where to find the database backup file (.bak) to restore from. If this option is not specified, no restore will be done.
```

Comment:
This one should be changed to just `--restore`, as the option name has been changed to that.

Reply:
I fixed it myself :)